### PR TITLE
Now with privacy page

### PR DESCRIPTION
**Pull request by meyerinj**

### CODE REVIEW

File: templates/document.html [hunk 1]
```diff
@@ -45,7 +45,7 @@
                 <li><a href='[%url]/download.html'>Download</a></li>
                 <li><a href='https://plugins.grails.org'>Plugins</a></li>
                 <li><a href='https://guides.grails.org/index.html'>Guides</a></li>
-                <li><a href='[%url]/foundation/index.html'>Foundation</a></li>
+                <li><a href='[%url]//foundation//index.html'>Foundation</a></li>
```
Comment (JasonTypesCodes) [inline templates/document.html:48]: Should this stay in?

File: pages/privacy-policy.html [hunk 2]
```diff
@@ -0,0 +1,340 @@
+---
+title: Community | Grails&reg; Framework
```
Comment (JasonTypesCodes) [inline pages/privacy-policy.html:2]: I think the title should be "Community | Grails&reg; Framework"

Comment (kroegerk) [reply pages/privacy-policy.html:2]: I think there's something that adds the pipe and the Grails® Framework part to all of the titles, but I could be mistaken.

**Review by kroegerk — APPROVED** [review]
As far as I can tell, this all looks correct and good to go. @meyerinj , thank you SO much for jumping in and getting this done so quickly. You're a rock star!

File: templates/document.html (hunk 1 above)
Comment (kroegerk) [reply templates/document.html:48]: I am unable to answer this one.

**Review by JasonTypesCodes — APPROVED** [review]

**Pull request merged**